sdk/formrecognizer/Azure.AI.FormRecognizer/src/FieldData.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+
+namespace Azure.AI.FormRecognizer.Models
+{
+    /// <summary>
+    /// The details of field elements that are part of a <see cref="FormField"/>.
+    /// </summary>
+    public class FieldData : FormElement
+    {
+        /// <summary>
+        /// </summary>
+        /// <param name="FormElement"></param>
+        /// <param name="pageNumber"></param>
+        /// <param name="boundingBox"></param>
+        /// <param name="text"></param>
+        internal FieldData(string text, int pageNumber, BoundingBox boundingBox, IReadOnlyList<FormElement> FormElement)
```

Comment:
nit: I know it's internal, but since it's a parameter name, `FormElement` => `formElement`.